.github/workflows/container.yml

```diff
@@ -52,7 +52,7 @@ jobs:
       - name: Build Image
         uses: docker/build-push-action@v5.0.0
         with:
-          context: .
+          context: backend/
```

Comment:
Shall we change the name of this job to `build-backend` to reflect the fact it's only for the backend containers?